### PR TITLE
fix water rendering in splitscreen

### PR DESCRIPTION
**Pull request by Berthalamew**
image

### CODE REVIEW

File: xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_water.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    if (!*water_reflection_rendered)
+    {
+        rasterizer_dx9_reset_depth_buffer();
+        if (get_global_structure_bsp()->water_definitions.count > 0)
+        {
+            IDirect3DSurface9* surface = rasterizer_dx9_target_get_surface(_rasterizer_target_20);
+            s_rasterizer_target* target_data = rasterizer_dx9_target_get(_rasterizer_target_20);
+
+            e_rasterizer_target present_target = *rasterizer_dx9_present_target_get();
```
Comment (nukeulater) [inline xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_water.cpp]: `rasterizer_dx9_main_render_target_get()` would be a better name

File: xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_water.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            IDirect3DSurface9* present_surface = rasterizer_dx9_get_target_surface((uint16)present_target, 0);
```
Comment (nukeulater) [inline xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_water.cpp]: `render_surface` and `get_render_target_surface`

File: xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_targets.h [hunk 3]
```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+/* enums */
+enum e_rasterizer_target : uint32
```
Comment (nukeulater) [inline xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_targets.h:4]: `e_rasterizer_render_targets` ?

Comment (Berthalamew) [reply xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_targets.h:4]: going to keep it as e_rasterizer_target

**Pull request merged**